webapp/src/app/core/IssueCard/issue-card.component.html

```diff
@@ -0,0 +1,37 @@
+<div class="border border-border bg-card rounded-lg p-4 w-72 cursor-pointer" (click)="openIssue()">
+  <div class="flex justify-between items-center mb-2 text-xs text-gray-500">
+    <span class="font-bold flex justify-center items-center space-x-1">
+      @if (state() === 'OPEN') {
+        <ng-icon [svg]="octGitPullRequest" size="16" class="mr-1 text-github-success-foreground"></ng-icon>
+      } @else {
+        <ng-icon [svg]="octGitPullRequestClosed" size="16" class="mr-1 text-github-danger-foreground"></ng-icon>
+      }
+
+      {{ repositoryName() }} #{{ number() }} on {{ createdAt().format('MMM D') }}
+    </span>
+    <span class="flex items-center space-x-2">
+      <span class="text-github-success-foreground font-bold">+{{ additions() }}</span>
+      <span class="text-github-danger-foreground font-bold">-{{ deletions() }}</span>
+    </span>
+  </div>
+
+  <div class="flex justify-between font-bold text-sm mb-3">
+    {{ title() }}
```

Comment:
If we remove the `(click)` and `cursor-pointer` from the main card `div` (not included in this suggestion) and move it to the title itself, we could create an interactive element like this, making it very similar to Github's own cards:
```suggestion
    <div class="hover:text-github-accent-foreground cursor-pointer" (click)="openIssue()">
      {{ title() }}
    </div>
```
Reference (on title hover):
![image](https://github.com/user-attachments/assets/7a88f8cb-27f2-48b9-bbf8-2245271fb720)